liquidctl/cli.py

```diff
@@ -191,6 +213,30 @@ def _list_devices(devices, using_filters=False, device_id=None, verbose=False, d
     assert 'device' not in opts or len(devices) <= 1, 'too many results listed with --device'
 
 
+def _dev_status_obj(dev, status):
+    # unlike the humans, machines want to know everything; don't suppress
+    # devices without status data (typically from initialize)
```

Comment:
Does this comment also mean that the json output should be equivalent to the verbose flag?

Reply:
I considered that, and opted to *not* imply `--verbose` in `status`/`initialize` since that enables output from more _devices._  But now I'm thinking that was probably a mistake.

Reply:
The current implementation does enable output from all devices. If i were to run `liquidctl status --json` I get output that includes my GPU, but the gpu is not listed from a normal `liquidctl status`. Based on your code that _seems_ intentional. imo, `--json` should imply verbose if the current implementation has all devices included in the list with no status output.

Reply:
Hm, it does, but the actual status are still suppressed at the driver level, which I (now) think is wrong.

Of course, in the case of I²C/SMBus devices `--unsafe <...>` will also be necessary to avoid the values from being suppressed.  But that's reasonable, I think.

Reply:
> Of course, in the case of I²C/SMBus devices --unsafe <...> will also be necessary to avoid the values from being suppressed. But that's reasonable, I think.

I agree with that part